sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Helpers/SymbolicExecution/TooManyInternalStatesException.cs

```diff
@@ -19,10 +19,26 @@
  */
 
 using System;
+using System.Runtime.Serialization;
 
 namespace SonarAnalyzer.Helpers.FlowAnalysis.Common
 {
+    [Serializable]

```

Comment:
I doubt this exception will be able to bubble up outside of the method where it is caught :)